client/src/document/on-github.tsx

```diff
@@ -39,7 +38,7 @@ function SourceOnGitHubLink({ doc }: { doc: Doc }) {
       target="_blank"
       rel="noopener noreferrer"
     >
-      Source on <b>GitHub</b>
+      View the source on <b>GitHub</b>
```

Comment:
How about "View its source on GitHub" or "View page source on GitHub" instead?

```suggestion
      View its source on <b>GitHub</b>
```

PS: Firefox omits the article in the context menu:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/495429/207554572-58aad96a-d343-4dd2-b2ef-e6f75291cc62.png">

Reply:
I guess that's because those are sub-menu items in the FF context menu and are worded as command names and follow title casing.
In the footer on our page, we are writing imperative sentences. (Ideally, those sentences should be closed off with periods.)